lessons/githubPages/lesson.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: page
+title: 'Web design with Jekyll/Github Pages'
+visible: false
+---
+<!-- change visible to true if you want it on the site -->
+
+ - **Authors**: Dawn Walker [@dcwalk](https://github.com/dcwalk)
+ - **Research field**: Information Studies (Information System Design)
+ - **Lesson topic**: Introduction to using Github Pages and Jekyll for website design
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/githubPages>
+
+## Preface
+This is a basic introduction to using Github Pages and Jekyll to host a web page. Github Pages are a feature of Github that allows users to host websites for individuals, projects, or organizations right from their repositories! The engine Github uses to accomplish this is Jekyll, a ["simple, static, blog-aware, site generator"](https://jekyllrb.com/), but it also supports regular HTML content.
+
+In this code-along we'll cover what Github Pages are, why you might want to use them, and how to do so both online and locally with some jekyll basics!
+
+### Pre-requisites
+
+- A github account (if you don't already have one go to [github.com/join](https://github.com/join))
+- git installed, and configured on your machine ([git-scm.com](https://git-scm.com/downloads) and UofT Coders lessons **[Intro to Git](https://uoftcoders.github.io/studyGroup/lessons/git/lesson/)** and **[More on Git](https://uoftcoders.github.io/studyGroup/lessons/git2/lesson/)**)
+
+### Bluebird Conditions (AKA would be great to have!)
+- jekyll and its requirements installed on your machine ([jekyllrb.com/docs/installation](https://jekyllrb.com/docs/installation/))
+  - ruby ([ruby-lang.org/en/documentation/installation](https://www.ruby-lang.org/en/documentation/installation/))
+  - rubygems ([rubygems.org/pages/download](https://rubygems.org/pages/download))
+  - some familiarity with **markdown** and **html**
```

Comment:
ruby and rubygems are requirements! (and were intended to be tabbed in.)

familiarity with markdon was intended to be top level, added a link!